tutorials/2d/introduction_to_2d.rst

```diff
@@ -0,0 +1,316 @@
+.. _doc_introduction_to_2d:
+
+Introduction to 2D
+==================
+
+Welcome to the exciting world of 2D game development with Godot! Whether you're a 
+seasoned developer or just starting out, Godot offers a friendly and intuitive environment 
+to bring your game ideas to life. With its powerful yet accessible features, you'll have 
+the freedom to unleash your creativity and create captivating gaming experiences. From 
+designing unique characters and crafting immersive worlds to implementing engaging 
+gameplay mechanics, Godot provides a comprehensive set of tools and resources to support 
+your game development journey. So, get ready to embark on an adventure filled with endless 
+possibilities and let your imagination soar as you dive into the wonderful realm of 2D game 
+development with Godot!
+
+.. figure:: img/2d_platformer_demo.webp
+   
+   2D Platformer Demo available on the Asset Library.
+
+This page will show you the 2D workspace and how you can get to know it.
+
+.. tip:: If you would like to get an introduction to 3D, see :ref:`doc_introduction_to_3d`.
+
+2D workspace
+~~~~~~~~~~~~
+
+You will use the 2D workspace to work with 2D scenes, design levels, or create user 
+interfaces.
+You can use the workspace selector located in Godot's top edge at the middle, or choose a 2D 
+related node from the scene tree to switch to the 2D editor:
```

Comment:
```suggestion
To switch to the 2D workspace, you can either select a 2D node from the scene tree, or use the workspace selector located at the top edge of the editor:
```
I think if we pick "2D workspace" as the term, it should be consistently used here? Also mildy reworded for clarity